From 644e704780ed5efb1def2bd67268ea6ea569d954 Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Mon, 7 Nov 2022 21:46:56 +0100
Subject: [PATCH] hwdb: Add accel orientation quirk for the Acer Switch V 10
 SW5-017 2-in-1

Add a quirk for the accelerometer orientation for
the Acer Switch V 10 SW5-017 2-in-1.

(cherry picked from commit 493f67d21d5552f0921334abfd39230c062c2d5a)

Related: RHEL-5950
---
 hwdb.d/60-sensor.hwdb | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index 000871537e..ea6b3f0452 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -92,6 +92,9 @@ sensor:modalias:acpi:INVN6500*:dmi:*svn*Acer*:*pn*AspireSW5-012:*
 sensor:modalias:acpi:BOSC0200*:dmi:*svnAcer*:*TP-SW5-017-17BU:*
  ACCEL_MOUNT_MATRIX=0, 1, 0; 1, 0, 0; 0, 0, -1
 
+sensor:modalias:acpi:BOSC0200*:dmi:*:svnAcer:pnSW5-017:*
+ ACCEL_MOUNT_MATRIX=0, 1, 0; 1, 0, 0; 0, 0, 1
+
 sensor:modalias:acpi:BMA250E*:dmi:*:svnAcer:pnIconiaW1-810:*
  ACCEL_MOUNT_MATRIX=1, 0, 0; 0, -1, 0; 0, 0, 1
 
